test/fixtures/external-modules/web-loader/css.mjs

```diff
@@ -0,0 +1,22 @@
+export async function resolve(specifier, ctx, next) {
+  const nextResult = await next(specifier, ctx);
+
+  if (!specifier.endsWith('.css')) return nextResult;
+
+  return {
+    format: 'css',
+    url: nextResult.url,
+  };
+}
+
+export async function load(url, ctx, next) {
+  const result = await next(url);
+
+  if (result.format !== 'css') return result;
+
+  return {
+    format: 'module',
+    shortCircuit: true,
+    source: `export default "${result.source} .test { color: red; }"`,
```

Comment:
We probably should use something other than CSS as an example. In browsers, `import styles from './styles.css' assert { type: 'css' }` works, and creates an instance of something (a `CSSStyleSheet` object or something like that). So we shouldn’t have an example of CSS that returns a string. Maybe change this to be supporting `.txt` or the like, as that would have no alternative to being loaded as a string.